pom.xml

```diff
@@ -87,7 +87,7 @@
         <solrj.version>1.4.1</solrj.version>
         <stax-utils.version>20070216</stax-utils.version>
         <spock.version>1.3-groovy-2.5</spock.version>
-        <shedlock.version>2.5.0</shedlock.version>
+        <shedlock.version>4.23.0</shedlock.version>
```

Comment:
Are there any breaking changes in this version change? Maybe update shedlock itself in a separate PR?

Reply:
The configuration is easier and easier to find documentation for, hence why I decided to upgrade as part of setting the job to run in AMS time. As per breaking changes, extensive testing would help answer that with surety.

Reply:
Reading the release notes here https://github.com/lukas-krecan/ShedLock#release-notes nothing jumps out. Via testing we know for sure

Reply:
Thanks